core-web/libs/portlets/edit-ema/portlet/src/lib/components/dot-ema-dialog/dot-ema-dialog.component.ts

```diff
@@ -314,4 +332,24 @@ export class DotEmaDialogComponent {
 
         this.action.emit({ event: customEvent, payload: this.dialogState().payload });
     }
+
+    /**
+     * Brings back the dialog to the previous state and triggers a reload event.
+     * @param options - The options for bringing back the dialog.
+     * @param options.name - The name of the dialog.
+     * @param options.args - The arguments for the dialog.
+     */
+    bringBack({ name, args }: { name: string; args: string[] } = { name: '', args: [] }) {
+        this.$compareData.set(null);
+        this.callEmbeddedFunction(name, args, () => this.reloadFromDialog.emit());
```

Comment:
You have access to the store, you can trigger a reload from here

Reply:
mm dialog have access to dialogStore, not to EditEmaStore
I think we should not give the component access to the editor-store

Reply:
Yup you are right